### PR TITLE
Correct Colour Palette for UI

### LINKED ISSUE #147: Add colour palette to UI

### PR DESCRIPTION
**Pull request by stedipace**

### CODE REVIEW

File: Assets/Prefabs/UI/OfficeTerminal/TerminalCanvas.prefab [hunk 1]
```diff
@@ -1551,10 +1551,10 @@ RectTransform:
   m_Father: {fileID: 3631385489712040968}
   m_RootOrder: 1
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
-  m_AnchorMin: {x: 0, y: 0}
```
Comment (maxwellpark) [inline Assets/Prefabs/UI/OfficeTerminal/TerminalCanvas.prefab]: You changed the anchors here. Was that intentional?

File: Assets/Prefabs/UI/OfficeTerminal/TerminalCanvas.prefab [hunk 2]
```diff
@@ -2042,10 +2042,10 @@ RectTransform:
   m_Father: {fileID: 3631385488924881247}
   m_RootOrder: 5
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
-  m_AnchorMin: {x: 0, y: 0}
-  m_AnchorMax: {x: 0, y: 0}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: 0, y: 0}
+  m_AnchorMin: {x: 0, y: 1}
```
Comment (maxwellpark) [inline Assets/Prefabs/UI/OfficeTerminal/TerminalCanvas.prefab]: Same here

**Review by maxwellpark — COMMENTED** [review]
You changed some anchors on the rect transform. Dunno if that was intentional

File: Assets/Prefabs/UI/OfficeTerminal/TerminalCanvas.prefab [hunk 3]
```diff
@@ -978,7 +978,7 @@ MonoBehaviour:
   reportCardPrefab: {fileID: 9116196081526937484, guid: cc5ea168ac6f73c4b9526ecca8c0ee9d,
     type: 3}
   reportCardInstance: {fileID: 5717576540381219605}
-  missionsPanel: {fileID: 2574278240523882435}
+  reportCard: {fileID: 0}
```
Comment (ambid17) [inline Assets/Prefabs/UI/OfficeTerminal/TerminalCanvas.prefab]: 🤔

**Pull request merged**